core/src/main/java/org/kiwix/kiwixmobile/core/CoreApp.java

```diff
@@ -138,7 +138,7 @@ private void writeLogFile() {
     if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR2) {
       builder.detectFileUriExposure();
     }
-    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN) {
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1) {
```

Comment:
I think you have gotten the wrong idea.

We were guarding this code because we needed to be at least `JELLY_BEAN` aka 17 to safely run it. Now we are always at 17 so we can always run it, it is not correct to just change the version code we are comparing